array-loops.js

```diff
@@ -3,6 +3,12 @@
 // --------------------------
 
 const names = ["Henk", "Piet", "Fred", "Joop"];
```

Comment:
Jouw manier werkt ook, maar als er nu 20 namen in de array hadden gezeten dan had je een hele hoop regels code moeten schrijven! Je had hier beter een for-loop kunnen gebruiken die via een if-statement achter iedere naam in de array "je, " had geplakt en alleen bij de laatste "je" erachter had geplakt.